### DISCUSSION TITLE
Ideas for Faceted Track Selector

### DISCUSSION THREAD

**Discussion by jbrestel** [GMOD/jbrowse-components#3853] [Ideas]
I've been thinking about some issues with the Faceted Track selector.  From our user experience, the faceted track selector in jbrowse1 was a bit hard for users to figure out.  Looks like the jbrowse2 one is based on that and maybe even not as intuitive.

I suggest the following:  
1.  Remove the boxes on the right which show the counts for each variable.  (Currently these do not update counts as users apply filters)
2. Make the table view better by making "show extra table options" more prevalent.  The table view let's you apply "filters" to column values which is exactly what we want.  NOTE:  Currently, not sure it is possible to apply multiple filters but that should be possible regardless.   This will give you a simple table format that users will feel comfortable with. 
3.  Use this same UX for subtrack selection.  (show metadata for subtracks in table which is filterable. )
4.  If seeing distributions/counts of variable values is important (like currently shown in the textArea boxes)... get to that from the table header when applying filters.

**Comment by cmdcolin** [comment]
thanks again for the detailed reports here (sorry for late response :). will definitely check into this. seeing the 'counts/distributions' of variables is sort of historically based on the "facets" of the faceted selector. the new faceted selector was admittedly pushed into production a little quick but it is great to get feedback from real users to help refine and make better.

**Reply by cmdcolin** [reply]
also, unfortunately, multi-filtering is not available at least 'out of the box'. we use something called the x-data-grid which has a 'premium plan' data grid widget that includes the multi-filtering, but we can't use that in our open source code

we could potentially re-implement multi-filtering ourselves, and indeed, the facets kind of are supposed to help do this (the boxes on the right)
